uint32 trans_seq
int32 trs_result
int32 reroute_index
